src/content/en/2022/javascript.md

```diff
@@ -6,7 +6,7 @@ description: JavaScript chapter of the 2022 Web Almanac covering the usage of Ja
 authors: [malchata, ibnesayeed]
 reviewers: [mgechev, pankajparkar, NishuGoel, seldo, housseindjirdeh, kevinfarrugia]
 analysts: [NishuGoel, kevinfarrugia]
-editors: [DesignrKnight]
+editors: [rviscomi]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1vOeFUyfEtWRen3Xj57ZsWav40n5tlcJoV0HmAxmNE_I/
 featured_quote: TODO
```

Comment:
@malchata can you suggest a brief quote and three of the most noteworthy stats from this chapter that we could use to feature it on our home page?

Reply:
Quote:

> "The state of JavaScript is a constantly evolving phenomenon. It's clear that we have an increased reliance on it more than ever, but that spells trouble for the collective user experience of the web. We need to do all we can—and _more_—to stem the tide of how much JavaScript we ship on production websites."

Stats:

1. 26% of the top million pages transform code with Babel.
2. Mobile devices download 8% more JavaScript in 2022 than in 2021.
3. Hosts at the 90th percentile ship 34 third-party scripts.

WDYT @rviscomi?

Reply:
Suggested ones:

```
featured_quote: The state of JavaScript is a constantly evolving phenomenon. It's clear that we have an increased reliance on it more than ever, but that spells trouble for the collective user experience of the web. We need to do all we can—and more—to stem the tide of how much JavaScript we ship on production websites.
featured_stat_1: 26%
featured_stat_label_1: Top million pages that transform code with Babel
featured_stat_2: 8%
featured_stat_label_2: Extra JavaScript downloaded in last year on mobile
featured_stat_3: 20
featured_stat_label_3: Median third-party scripts shipped
```

Changes from your suggestions:
- Remove unnecessary italics from quote (we try to avoid over using them in the Web Almanac)
- Rephrase stats to fit in stat/label format
- Suggest using median rather than 90th percentile for featured stats.

Reply:
Works for me!